src/console/CliManager.cpp

```diff
@@ -142,15 +144,28 @@ void CliManager::loop() {
         }
         // Line break
         while (!quit && !line.empty() && line.back() == '\\') {
-            line.resize(line.size() - 1);
+            auto lineSize = line.size() - 1; 
+            if (lineSize > kMaxCmdLen) {
+                cmdTooLarge = true; 
+                break;
+            }
+            line.resize(lineSize);
             cmd += line;
+            if(cmd.length() > kMaxCmdLen) {
+                cmdTooLarge = true; 
+                break;
+            }
```

Comment:
Only catch the case there are line breaks?
I think you could just check right before the command gets executed/processed.